api/envoy/type/v3/udp_route_policy.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package envoy.type.v3;
+
+import "envoy/type/v3/hash_policy.proto";
+
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.type.v3";
+option java_outer_classname = "UdpRoutePolicyProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: UDP Route Policy]
+
+// Specifies the udp route policy.
+message UdpRoutePolicy {
```

Comment:
I would just inline this message inside udp_proxy.proto. I don't think it needs its own file and also shouldn't be in the types directory.

Reply:
By my understanding, this message will be used by QUIC too. So, I intend to put it into more general place. Isn't it? Please let me know if I'm wrong. :)

Reply:
It won't be used by QUIC, but by L4 UDP proxy code that might route QUIC packets. So I think keeping it in the UDP proxy config is fine.

Reply:
Ok. It is caused by my lack of understanding of QUIC. I got your point. :)